packages/mobile/src/app/actions.ts

```diff
@@ -194,10 +197,11 @@ export const setSessionId = (sessionId: string) => ({
   sessionId,
 })
 
-export const openUrl = (url: string, openExternal: boolean = false): OpenUrlAction => ({
+export const openUrl = (url: string, openExternal = false, isSecureOrigin = false): OpenUrlAction => ({
```

Comment:
Nothing to do now, having boolean flags as direct arguments makes it quite hard to understand what it does when seeing the function call, for instance `openUrl(anUrl, true, false)`.

I know this has been discussed in the past and we landed on avoiding using arguments in objects
https://github.com/celo-org/celo-monorepo/blob/master/packages/docs/celo-codebase/style-guide/typescript.md
But I think for cases like here it shouldn't apply.

Here's what I'd prefer `openUrl(anUrl, { openExternal: true, isSecureOrigin: false })`